test/controllers/wedding_party_members_controller_test.rb

```diff
@@ -16,8 +16,8 @@ class WeddingPartyMembersControllerTest < ActionDispatch::IntegrationTest
   end
 
   test "should create wedding_party_member" do
-    assert_difference('WeddingPartyMember.count') do
-      post wedding_party_members_url, params: { wedding_party_member: { firstName: @wedding_party_member.firstName, lastName: @wedding_party_member.lastName, project_id: @wedding_party_member.project_id, relationship: @wedding_party_member.relationship, relationshipTo: @wedding_party_member.relationshipTo, role: @wedding_party_member.role } }
+    assert_difference("WeddingPartyMember.count") do
+      post wedding_party_members_url, params: { wedding_party_member: { namePrefix: @wedding_party_member.namePrefix, firstName: @wedding_party_member.firstName, lastName: @wedding_party_member.lastName, project_id: @wedding_party_member.project_id, role: @wedding_party_member.role, relationship: @wedding_party_member.relationship, relationshipTo: @wedding_party_member.relationshipTo } }
     end
 
     assert_redirected_to wedding_party_member_url(WeddingPartyMember.last)
```

Comment:
Change this line to:

```ruby
assert_redirected_to wedding_party_member_url(WeddingPartyMember.order(:created_at).last)
```

If you don't specify an order, the database will typically return records ordered by the primary key (ID in this case). That works fine when the ID is an auto-inc, but it looks like these IDs are now GUIDs. As such, `WeddingPartyMember.last` is returning the highest GUID (which is a meaningless order). Specifying `WeddingPartyMember.order(:created_at).last` returns the most recently created row, which is what test intends.

Reply:
got it! thanks for sending me in the right direction, I see what's happening now.